DungeonGame.py

```diff
@@ -242,34 +248,31 @@ def entrance_hall():
     rdoor = goblin_room()
 
     staff = Item(
-    name = "The Staff of Power"
-    description = ("The Staff is tall and twisted, made of a deep dark wood and topped with an ever changing crystal.",
-                    " It exudes power, pulsing and searing against the air.")
-    take_me = "The torches blaze up, wind howles through the room, and lightning strikes the crystal atop it! You got The Staff of Power!"
-    valid_choices = ("Staff", "staff", "A Staff", "a Staff", "A staff", "a staff", "The Staff",
-    "the Staff", "The staff", "the staff")
+        name = "The Staff of Power",
+        description = "The Staff is tall and twisted, made of a deep dark wood and topped with an ever changing crystal. It exudes power, pulsing and searing against the air.",
+        take_me = "The torches blaze up, wind howles through the room, and lightning strikes the crystal atop it! You got The Staff of Power!",
+        valid_choices = ("Staff", "staff", "A Staff", "a Staff", "A staff", "a staff", "The Staff", "the Staff", "The staff", "the staff"),
     )
-    game_state.room.keys["staff"] = staff
+
+    game_state['room']['items'].append(staff)
```

Comment:
`game_state['room']['items']` is a list, so we have to `append()` to it instead of trying to set the `['staff']` key -- lists don't have keys.